rim/views.py

```diff
@@ -126,3 +143,14 @@ class EditEquipmentView(UpdateView):
     model = Equipment
     form_class = EquipmentForm
     success_url = reverse_lazy('home')
+
+class ClientView(DetailView):
+    template_name = 'rim/client.html'
+    model = Client
+
+    def get_context_data(self, *args, **kwargs):
+        context = super(ClientView, self).get_context_data(*args, **kwargs)
+        context['client'] = Client.objects.get(pk=self.kwargs['pk'])
```

Comment:
Believe it or not, Django is already filling in `context['client']` for you since this is a [DetailView](https://ccbv.co.uk/projects/Django/2.0/django.views.generic.detail/DetailView/). So there's no reason to redefine it yourself.

Reply:
fixed